.github/workflows/create-release.yml

```diff
@@ -172,20 +200,28 @@ jobs:
         name: current-build-image
         path: build/build.oci
 
-    - name: Upload Build receipt
+    - name: Generate Package Receipts
+      id: receipts
+      run: |
+        scripts/receipts.sh --build-image "${GITHUB_WORKSPACE}/build/build.oci" \
+                            --run-image "${GITHUB_WORKSPACE}/build/run.oci" \
+                            --build-receipt ${{ env.BUILD_RECEIPT_FILENAME }} \
+                            --run-receipt ${{ env.RUN_RECEIPT_FILENAME }}
+
+    - name: Upload ${{ matrix.arch }} Build receipt(s)
       uses: actions/upload-artifact@v3
       with:
         name: current-build-receipt
-        path: ${{ steps.receipts.outputs.build_receipt }}
+        path: "*${{ env.BUILD_RECEIPT_FILENAME }}"
```

Comment:
I see how the `*` works, but why do we need it? It looks like above we just write the file to: `--build-receipt ${{ env.BUILD_RECEIPT_FILENAME }}`

Reply:
The output of the Generate Package Receipts step file will be `<ARCH>-${{ env.BUILD_RECEIPT_FILENAME }}`, so in this case we will have `amd64-${{ BUILD_RECEIPT }}` and `arm64-${{ BUILD_RECEIPT }}`, so we need to upload both, which we can do using the `*`.

Reply:
the `--build-receipt ${{ env.BUILD_RECEIPT_FILENAME }}` is just the file base name, but for all the files we generate we have an arch-specific version (receipts, release notes, usn file)

Reply:
ah ok. makes sense - thanks 👍